src/ast/mod.rs

```diff
@@ -5807,6 +5807,11 @@ pub struct Function {
     /// <aggregate_function>(expression) WITHIN GROUP (ORDER BY key [ASC | DESC], ...)
     /// ```
     pub within_group: Vec<OrderByExpr>,
+
+    /// Sometimes function names may also be column names,
+    /// for example, user, current_user,
+    /// it requires the user to determine whether it is a column name or a function name.
+    pub maybe_column: bool,
```

Comment:
I think the desired behavior is that the keywords `USER`, `CURRENT_USER` etc are represented as identifiers in the scenarios where they are not functions. Having a flag on the function saying that the flag is in fact not a function sounds like it would be misleading/confusing behavior to the user